### ISSUE TITLE
No horizontal scroll bar in editor when wrap line is OFF in windows version

### ISSUE THREAD

**Issue by guruathwal** [BoostIO/Boostnote#3628]
# Current behavior
In the windows version, The horizontal scroll bar in the editor is not visible if long lines are added with wrap lines off 

# Expected behavior

The horizontal scroll bar should be visible if the editor has a longer line with the wrap lines turned off.

# Steps to reproduce
1.  turn off wrap line.

2.  add a long line without line break like this one:
`Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.`

2. the horizontal scroll bar is not visible. 

# Environment

- Boostnote version: Boostnote Legacy 0.16.0
- OS version and name: Windows 10

**Comment by Flexo013** [comment]
As a work-around for now you can scroll the code block by selecting the text.

**Comment by guruathwal** [comment]
Yes, doing the same but some times it is hard to identify the longer lines.
